client/my-sites/upgrades/cart/test/test-cart-buttons.jsx

```diff
@@ -3,7 +3,7 @@ require( 'lib/react-test-env-setup' )();
 /**
  * External Dependencies
  */
-var	sinon = require( 'sinon' ),
+const	sinon = require( 'sinon' ),
```

Comment:
Does this change need to be made? If we're changing things, I'd rather see this go to `import` syntax than to `const`.